src/talkaction.cpp

```diff
@@ -61,7 +61,8 @@ Event* TalkActions::getEvent(const std::string& nodeName)
 
 bool TalkActions::registerEvent(Event* event, const pugi::xml_node&)
 {
-	talkActions.push_front(*static_cast<TalkAction*>(event)); // event is guaranteed to be a TalkAction
+	auto talkAction = std::unique_ptr<TalkAction>(static_cast<TalkAction*>(event)); // event is guaranteed to be a TalkAction
+	talkActions.push_front(std::move(*talkAction.get()));
```

Comment:
Just noticed you can use `*talkAction`, `.get()` is redundant [(source)](http://en.cppreference.com/w/cpp/memory/unique_ptr/operator*)